src/cosalib/container_manifest.py

```diff
@@ -40,23 +41,23 @@ def push_container_manifest(repo, main_tag, add_tags, digestfile, v2s2=False):
         # to create a manifest with 2 different mediaType. It seems to be
         # a Quay issue.
         base_cmd.extend(["--remove-signatures", "-f", "v2s2"])
-    if digestfile:
-        base_cmd.extend([f"--digestfile={digestfile}"])
     runcmd(base_cmd + [f"{repo}:{main_tag}"])
     for tag in add_tags:
         runcmd(base_cmd + [f"{repo}:{tag}"])
 
 
-def create_and_push_container_manifest(repo, tags, images, v2s2, digestfile):
+def create_and_push_container_manifest(repo, tags, images, v2s2):
     '''
     Do it all! Create, Push, Cleanup
     @param repo str registry repository
     @param tags list of tags
     @param images list of image specifications (including transport)
     @param v2s2 boolean use to force v2s2 format
-    @param digestfile str write container digest to file
     '''
     main_tag = tags[0]
     create_local_container_manifest(repo, main_tag, images)
-    push_container_manifest(repo, main_tag, tags[1:], digestfile, v2s2)
+    push_container_manifest(repo, main_tag, tags[1:], v2s2)
+    inspect = runcmd(["podman", "manifest", "inspect", f"{repo}:{main_tag}"],
+                     capture_output=True).stdout
```

Comment:
Does this inspect the `local` container manifest that was just created? Depending on the answer I think we should do one of two things:

- `YES` - then let's move this above the `push_container_manifest()` call to make it obvious it's just inspecting the local manifest that was just created. We also need to document this better and maybe annotate the function to indicate it's now returning a value.
- `NO` - then let's move this `podman manifest inspect` call to the caller (`cmd-push-container-manifest`).

Reply:
I reworked this. Technically, `create_local_container_manifest` should return that info, so I lowered it down to there. Documented it.